web/themes/joinup/templates/content/page-title--node--discussion.html.twig

```diff
@@ -17,7 +17,7 @@
   <div class="page__title-wrapper">
       <h1 class="page__title">{{ title }}</h1>
       {% if permalink %}
-          <a class="permalink" href="{{ permalink }}" title="Permalink">#</a>
+        <a class="permalink" href="{{ permalink }}" title="Permalink"><span class="icon icon--anchor"></span></a>
```

Comment:
I think it will be good to include some text in the link for screen readers in addition to the icon.